<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QWidget" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="TWDeposit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>471</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>step</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>start sum</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>adding sum</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>removing sum</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>sum</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>rezult sum</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>percents</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Final sum on account</string>
   </property>
  </widget>
  <widget class="QLabel" name="FinalSumOnAccaunt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>500</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total sum of percents</string>
   </property>
  </widget>
  <widget class="QLabel" name="TotalSumOfPercents">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>520</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>500</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax sum for percents (13%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="TaxSumForPercents">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>520</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
